src/RequestCommand.php

```diff
@@ -106,4 +107,22 @@ protected function parseContentType($type) {
 
         return (in_array($type, array_keys($types))) ? $types[$type] : (string) $type;
     }
+
+    /**
+     * Format the response based on which http verb you are using
+     * When you create an object through the rest API, Magento doesn't return a valid response and instead sets a
+     * "location" header
+     *
+     * @param $response
+     * @param $httpVerb
+     * @return string
+     */
+    protected function formatResponse($response, $httpVerb)
```

Comment:
I like what this does, but I think there's a smarter way of doing it.  What we're doing here is making a decision on the formatting of the response based on a parameter of the request. The info you need is in the response itself.

Instead of using the httpVerb, I'd prefer to check for the presence of a location header in the response.